parsers/src/main/scala/uk/gov/ons/addressIndex/parsers/Tokens.scala

```diff
@@ -387,12 +390,29 @@ object Tokens extends CrfTokenable {
     val directory = config.getString("parser.input-pre-post-processing.folder")
     val path = directory + fileName
     val currentDirectory = new java.io.File(".").getCanonicalPath
+    // `Source.fromFile` needs an absolute path to the file, and current directory depends on where sbt was lauched
+    // `getResource` may return null, that's why we wrap it into an `Option`
+    val resource = Option(getClass.getResource(path)).map(Source.fromURL).getOrElse(Source.fromFile(currentDirectory + path))
+    resource.getLines().toList
+  }
 
+  /**
+    * Make score matrix file into Map
+    * @param path
+    * @return
+    */
+  def fileToMap(fileName: String): Map[String,String] = {
+
+    val directory = config.getString("parser.scoring.folder")
+    val path = directory + fileName
+    val currentDirectory = new java.io.File(".").getCanonicalPath
     // `Source.fromFile` needs an absolute path to the file, and current directory depends on where sbt was lauched
     // `getResource` may return null, that's why we wrap it into an `Option`
     val resource = Option(getClass.getResource(path)).map(Source.fromURL).getOrElse(Source.fromFile(currentDirectory + path))
 
-    resource.getLines().toList
+    resource.getLines().map { l =>
```

Comment:
This is specific to the scoreMatrix,
the `fileToMap` method is used for other things, can you move this specific code to the line 366 so that it is not executed for other files?